examples/threathunter/threat_intelligence/stix_parse.py

```diff
@@ -207,11 +216,44 @@ def cybox_parse_observable(observable, indicator, timestamp, score):
     elif type(props) == File:
```

Comment:
Should this be `if` and not `elif`?

Reply:
@dly-cb You missed a bit of context. The if starts at line 209.

Reply:
Ah I didn't see the expansion on the left hand side. All good @llyon-cb